src/dials/command_line/import_xds.py

```diff
@@ -91,6 +91,7 @@ def __call__(self, params, options):
             table["miller_index"] = flex.miller_index(miller_index)
         table["xyzobs.px.value"] = flex.vec3_double(centroid)
         table["intensity.sum.value"] = flex.double(intensity)
+        table.set_flags(flex.bool(len(table), True), table.flags.strong)
```

Comment:
What is the motivation for setting all as strong?

Reply:
`SPOT.XDS` is supposed to be for spot-finding results, thus all strong. The issue #2960 pointed out that XDS chokes when an intensity is < 1, which we were hitting before by exporting even weak reflections in this format